csharp/src/Ice/UniversalEndpoint.cs

```diff
@@ -87,6 +81,12 @@ protected internal override void AppendOptions(StringBuilder sb, char optionSepa
             }
         }
 
+        protected internal override Endpoint Clone(string host) =>
```

Comment:
same here I think this can just throw

Reply:
Added a TODO comment.

Reply:
Removed TODO and replaced Clone by GetPublishedEndpoint that throws NotSupportedException as suggested by Jose/Joe.